contracts/strategies/strategy.sol.example

```diff
@@ -0,0 +1,125 @@
+/// @notice SPDX-License-Identifier: MIT
+pragma solidity ^0.6.7;
+
+/// @dev Import the appropriate base contract
+import "../strategy-joe-rush-farm-base.sol";
+
+/// @dev Contracts use NatSpec Format comments. NatSpec documentation is located here: https://docs.soliditylang.org/en/develop/natspec-format.html
+/// @notice The strategy contract for TraderJoe's Token1/Token2 Liquidity Pool with double rewards
+/// @dev If one of the tokens is a bridge extension token (ends with ".e"), the trailing E should be capitalized like so: StrategyJoeAvaxDaiE
+contract StrategyJoeToken1Token2 is StrategyJoeRushFarmBase {
+    /// @dev LP and Token addresses
+    uint256 public token1_token2_poolId = 10;
+    address public joe_token1_token2_lp = 0x1000000000;
+    
+    /// @dev Addresses for tokens not found in the base contracts
+    address public token2 = 0x2000000000;
+    address public reward1 = 0x300000000;
+
+    constructor(
+        address _governance,
+        address _strategist,
+        address _controller,
+        address _timelock
+    )
+        public
+        StrategyJoeRushFarmBase(
+            token1_token2_poolId,
+            joe_token1_token2_lp,
+            _governance,
+            _strategist,
+            _controller,
+            _timelock
+        )
+    {}
+
+    /// @dev Collect token fees and add liquidity to base pair
+    function harvest() public override onlyBenevolent {
+        /// @dev Ensure the deposit function interfaces with the correct controller for your platform (Joe, Pangolin, etc.)
+        IMasterChefJoeV2(masterChefJoeV3).deposit(poolId, 0);
+        
+        /// @dev Get balance of native AVAX and wrap AVAX into ERC20 (WAVAX)
+        /// @dev This function is necessary whenever dealing with AVAX
+        uint256 _avax = address(this).balance;
+        if (_avax > 0) {                                    
+            WAVAX(wavax).deposit{value: _avax}();
+        }
+        
+        /// @dev Check token balances, take fee for each token, then update balances
+        /// @dev The contract will require one balanceOf() call and one if() block per token
+        /// @dev Example: the JoeAvaxSavax contract has 4 tokens; AVAX, sAVAX, QI, and JOE
+        uint256 _token1 = IERC20(token1).balanceOf(address(this));
+        
+        if (_token1 > 0) {
+            uint256 _keep = _token1.mul(keep).div(keepMax);
+            if (_keep > 0){
+                /// @dev Check the imported base contract and strategy-base.sol contracts for the appropriate
+                /// @dev fee function for the token. Tokens may reqire a specific path to exchange for SNOB
+                _takeFeeWavaxToSnob(_keep);
+            }
+            
+            _token1 = IERC20(wavax).balanceOf(address(this));
```

Comment:
Instead of assigning quantities of different tokens to _token1, use separate local variables.